node/node_handler.go

```diff
@@ -410,6 +410,23 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block, commitSigAndBit
 // AddNewBlock is usedd to add new block into the blockchain.
 func (node *Node) AddNewBlock(newBlock *types.Block) error {
 	_, err := node.Blockchain().InsertChain([]*types.Block{newBlock}, true /* verifyHeaders */)
+
+	// Debug only
```

Comment:
please remove debug code.

Reply:
I will comment it out. It will be useful later